eo-runtime/src/main/eo/org/eolang/nan.eo

```diff
@@ -89,7 +97,7 @@
 
   # Converts this to bytes
   [] > as-bytes
-    7F-C0-00-00 > @
+    ^.@.as-bytes > @
```

Comment:
@maxonfjvipon I don't get this part. Can you provide some short explanation?

Reply:
`7F-C0-00-00` - is float NaN (4 bytes). 
On the last meeting we decided that float in `eo` is a double in Java - 8 bytes. 
Double NaN of 8 bytes - `7F-F8-00-00-00-00-00-00` (this value also will be got as result of `0.0.div 0.0`)

I tried to hardcode this bytes values and tests passed on my machine, but they fails in CI for some reason (you can check one of the previous commits)

So (0.0.div 0.0).as-bytes is kinda safe

Reply:
@maxonfjvipon I agree, thanks!